src/service_client/assisted_service_api.py

```diff
@@ -161,7 +161,7 @@ def get_all_clusters(self) -> List[Dict[str, Any]]:
     def cluster_get(self, cluster_id: str, get_unregistered_clusters: bool = False) -> models.cluster.Cluster:
         return self.client.v2_get_cluster(cluster_id=cluster_id, get_unregistered_clusters=get_unregistered_clusters)
 
-    def get_infra_env_by_cluster_id(self, cluster_id: str) -> List[Union[models.infra_env.InfraEnv, Dict[str, Any]]]:
+    def get_infra_envs_by_cluster_id(self, cluster_id: str) -> List[Union[models.infra_env.InfraEnv, Dict[str, Any]]]:
```

Comment:
QE doesn't seem to use ``get_infra_env_by_cluster_id``, so it should be ok to change